src/test/hydro/cache/test_in_memory.py

```diff
@@ -1,11 +1,14 @@
-__author__ = 'moshebasanchig'
-
 import unittest
-from hydro.cache.in_memory import InMemoryCache
+from django.conf import settings
+
+__author__ = 'moshebasanchig'
 
 
 class InMemoryCacheTest(unittest.TestCase):
     def setUp(self):
+        if not settings.configured:
+            settings.configure()
+        from hydro.cache.in_memory import InMemoryCache
```

Comment:
Yes, `in_memory` imports django's in memory cache class which will fail unless django was initialized beforehand, hence, we have to perform that import only after `settings.configure()`. I'll add a comment in the code.